feature-libs/order/root/model/order.model.ts

```diff
@@ -79,6 +80,8 @@ export interface Consignment {
   status?: string;
   statusDate?: Date;
   trackingID?: string;
+  //myaccount-enhanced-ui related
+  consignmentTracking?: ConsignmentTracking;
```

Comment:
[MAJOR]
As discussed offline, let's create a separate View Model for adding `consignmentTracking`.